packages/driver/cypress/integration/commands/assertions_spec.js

```diff
@@ -1277,6 +1277,36 @@ describe('src/cy/commands/assertions', () => {
     })
   })
 
+  // TODO: this suite should be merged with the suite above
```

Comment:
`TODO` is here because the test codes should be refactored a bit later. 

The suite above `format quotation marks` actually has 3 subcategories:

* format quotation marks
* format strings
* escape markdown

The name doesn't explain well about the tests. 

It'll be handled after this PR is merged.